docs/griptape-cloud/knowledge-bases/create-knowledge-base.md

```diff
@@ -0,0 +1,7 @@
+# Knowledge Bases
+
+Use [Knowledge Bases](https://cloud.griptape.ai/knowledge-bases) to prep your data ingested from [Data Sources](../data-sources/create-data-source.md) for retrieval. You can specify multiple Data Sources per Knowledge Base in order to access data ingested from different sources all in one place.
+
+## Create a Knowledge Base
+
+You can [create a Knowledge Base](https://cloud.griptape.ai/knowledge-bases/create) by specifying which Data Sources you wish to include. Once created, you can [access your data](accessing-data.md) instantly.
```

Comment:
"access your data instantly" can be a little misleading, as if I'm going through the process and setup a Data Source against a big source, and then immediately go create a KB with that DS, I need to wait for the Data Source to complete.

Reply:
Removed instantly.